engine/src/main/scala/org/scalarules/dsl/nl/ScalaRulesDslTrait.scala

```diff
@@ -1,16 +1,19 @@
 package org.scalarules.dsl.nl
 
 import org.scalarules.dsl.nl.datum.DatumImplicits
-import org.scalarules.dsl.nl.grammar.{DslCondition, DslConditionImplicits, DslEvaluationImplicits, GegevenWord}
+import org.scalarules.dsl.nl.grammar.{DslCondition, DslConditionImplicits, DslEvaluationImplicits, DslListFilterWord, DslLoopWordTrait, GegevenWord}
 import org.scalarules.engine._
 
 /**
   * Aggregates the keywords and implicit definitions of the Scala-Rules DSL. The implicits available in this
   * trait can be used by importing the `grammar` package object's members, or extending this trait.
   */
-trait ScalaRulesDsl extends DslConditionImplicits
+trait ScalaRulesDslTrait extends DslConditionImplicits
```

Comment:
I don't know. this must have snuck in there by accident. Your way seems better for sure